### PR TITLE
Part five two

### PR DESCRIPTION
**Pull request by swalsh15**
Fetches comments from datastore and displays in html. Added private class in servlet to send username and comment to html. Formats comments on page whenever windows loads - removed fetch comments button. 

Screenshot 2020-06-02 at 11 19 58 AM - Display 2

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -23,21 +28,52 @@
...
+    PreparedQuery results = datastore.prepare(query);
+    ArrayList<Comment> comments = new ArrayList();
 
-    Gson gson = new Gson();
-    String json = gson.toJson(list);
+    for (Entity entity : results.asIterable()) {
+      String comment = (String) entity.getProperty("title");
+      String user = (String) entity.getProperty("description");
+      Comment c = new Comment(user, comment);
+      comments.add(c);
```
Comment (Danish21) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:55]: Uber nit.
Consider inlining new comment:
```suggestion
      comments.add(new Comment(user, comment));
```
The local variable is not really used and the suggestion above is also not very long in terms of line length.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -23,21 +28,52 @@
...
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String comment = request.getParameter("text_input");
+    String name = request.getParameter("name_input");
+    if (comment == null || name == null) {
+      response.sendRedirect("/index.html");
```
Comment (Danish21) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:69]: Return after response.sendRedirect here so you do not add the comment to the datastore.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -39,17 +39,28 @@ function highlightTab() {
...
 }
 
-function getDataFromServlet() {
-  fetch('/data').then(response => response.text()).then((quote) => {
-    document.getElementById('data-container').innerText = quote;
-  });
+function fetchComments() {
+  (fetch('/comments').then(response => response.json()).then((comments) => {
+    for (const comment of comments) {
+      console.log(comment);
```
Comment (Danish21) [inline portfolio/src/main/webapp/script.js:45]: Remove console.log before merging.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -39,17 +39,28 @@ function highlightTab() {
...
+      let div = document.createElement("div");
```
Comment (augerm) [inline portfolio/src/main/webapp/script.js:46]: let --> const for div, user and message.

**Pull request merged**